includes/namespace.php

```diff
@@ -85,7 +85,8 @@ function rewrite_flush() {
  *
  * @return void
  */
-function activate( bool $network_wide = false ) {
+function activate( $network_wide = false ) {
```

Comment:
Since it [apparently](https://github.com/google/web-stories-wp/pull/8264#issuecomment-878076271) can be `null`, let's update `@param bool` to `@param bool|null`

Reply:
Fixed.